uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Helper/Contact.php

```diff
@@ -13,19 +13,19 @@ class CRM_HRContactActionsMenu_Helper_Contact {
    * @return array
    */
   public static function getUserInformation($contactID) {
-    $result = civicrm_api3('User', 'getsingle', [
-      'contact_id' => $contactID,
-    ]);
+    try {
+      $result = civicrm_api3('User', 'getsingle', [
+        'contact_id' => $contactID,
+      ]);
 
-    //When a user has no CMS account, the contact_id parameter is not present
-    //we need to add it here.
-    //Also when contact has CMS account, the cmsId parameter is needed for the
-    //HRCore CMSData classes
-    if(empty($result['id'])) {
-      $result['contact_id'] = $contactID;
-    }
-    else {
+      //Also when contact has CMS account, the cmsId parameter is needed for the
+      //HRCore CMSData classes
       $result['cmsId'] = $result['id'];
+    } catch(Exception $e) {
+      //When a user has no CMS account, the contact_id parameter is not present
+      //and an exception is thrown, we need to add it here.
+      $result['contact_id'] = $contactID;
+      unset($result['error_message']);
```

Comment:
what about the `is_error` property? Is it also present in the response? Tbh, I don't think that reusing the `$result` array here is a good idea. It's safer to simply create the array with the values we need